job/storage/mongo/mongo.go

```diff
@@ -0,0 +1,88 @@
+package mongo
+
+import (
+	"github.com/ajvb/kala/job"
+
+	log "github.com/Sirupsen/logrus"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var database = "kala"
+var collection = "jobs"
```

Comment:
@gwoo You could do

```golang
const (
    database = "kala"
    collection = "jobs"
)
```

Reply:
@elimisteve I don't think I want to make them constants because we do change them in the in the tests. Originally I had them as variables on New(), but that seemed like overkill. Maybe we have another method on the DB struct that changes them?